app/grants/models.py

```diff
@@ -314,34 +325,12 @@ class ContributionQuerySet(models.QuerySet):
 class Contribution(SuperModel):
     """Define the structure of a subscription agreement."""
 
-    tx_id = models.CharField(max_length=255, default='0x0', help_text=_('The transaction ID of the Contribution.'))
-    from_address = models.CharField(
+    tx_id = models.CharField(
         max_length=255,
         default='0x0',
-        help_text=_('The wallet address tokens are sent from.'),
-    )
```

Comment:
you just need to `./manage.py makemigrations grants` on this so it creates a migration

Reply:
will do